baybe/surrogates/_adapter.py

```diff
@@ -41,6 +39,12 @@ def posterior(  # noqa: D102
         **kwargs: Any,
     ) -> Posterior:
         # See base class.
-        mean, var = self._surrogate.posterior(X)
-        mvn = gpytorch.distributions.MultivariateNormal(mean, var)
-        return GPyTorchPosterior(mvn)
+        if (
```

Comment:
I do not get this check - the arguments are not used in the call to the private function, why check for them here? Is this just for having a clear signature in subclasses or am I simply missing something obvious here?

Reply:
Makes mucho sense